plugin-server/src/worker/ingestion/process-event.ts

```
@@ -249,6 +249,7 @@ export class EventsProcessor {
     ): Promise<void> {
         const personFound = await this.db.fetchPerson(teamId, distinctId)
         if (!personFound) {
+            this.pluginsServer.statsd?.increment('person_not_found', { teamId: String(teamId), key: 'update' })
             throw new Error(
                 `Could not find person with distinct id "${distinctId}" in team "${teamId}" to update properties`
             )
```

Why not stop throwing error in this case instead of ignoring it a higher level?